cfg/config_test.go

```diff
@@ -0,0 +1,77 @@
+package cfg
+
+import (
+	"github.com/google/go-cmp/cmp"
+	"os"
+	"testing"
+)
+
+func TestEnvVars(t *testing.T) {
+
+	defer unsetEnv()
+
+	os.Setenv("IMAGES", "che-server=eclipse/che-server:nightly")
```

Comment:
maybe let's change this for che-theia (no one is going to cache che-server images)